src/Containers/AddTodo/AddTodo.js

```diff
@@ -1,44 +1,110 @@
-import React from 'react';
-import { TextField } from '@material-ui/core';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+    TextField,
+    FormControl,
+    FormLabel,
+    RadioGroup,
+    FormControlLabel,
+    Radio,
+} from '@material-ui/core';
 import Grid from '@material-ui/core/Grid';
+
 import { connect } from 'react-redux';
+import { addTodo } from './Action';
 
-import { addTodo } from '../../actions';
 import Button from '../../Components/Button/Button';
 
+class AddTodo extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            priority: '',
+            input: '',
+        };
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.handleInput = this.handleInput.bind(this);
+    }
+
+    handleSubmit(event) {
+        const { input, priority } = this.state;
+        if (!input.trim()) {
+            return;
+        }
+
+        this.setState({ input: event.target.value });
```

Comment:
2 setState em seguida e ele não deu warning?

Reply:
Testei e não apresenta warning nem nenhum problema, mas acredito que isso leva a um estado não esperado do componente (setStates são assincronos). Como opções temos colocar no final da função o setState ou transforma-los em hooks.

Reply:
Removi o this.setState({ input: event.target.value }); pois o input é setado no onChange. No handleSubmit deixei apenas o setState que limpa o input.